### PR TITLE
Fix deprecation warnings.

### PR DESCRIPTION
**Pull request by icemac**
Fixed the ones which have shown up when running the tests in the way the warning suggested it.

### CODE REVIEW

File: src/AccessControl/userfolder.py [hunk 1]
```diff
@@ -111,7 +114,7 @@ def _doDelUsers(self, names):
     def identify(self, auth):
         if auth and auth.lower().startswith('basic '):
             try:
-                name, password = decodestring(auth.split(' ')[-1].encode()).decode() \
+                name, password = decodebytes(auth.split(' ')[-1].encode()).decode() \
```
Comment (tseaver) [inline src/AccessControl/userfolder.py]: We just compared `auth` against native str above:  are you confident it is really bytes?

Comment (icemac) [reply src/AccessControl/userfolder.py]: According to the documentation (https://docs.python.org/3/library/base64.html#base64.decodestring) `base64.decodestring` is a deprecated alias of `decodebytes`. I think at least my change does not make the code worse.

**Comment by icemac** [comment]
Green Travis CI depends on merging #55 into this branch.

File: src/AccessControl/userfolder.py (hunk 1 above)
Comment (tseaver) [reply src/AccessControl/userfolder.py]: OK, but if the Python3 stdlib is expecting bytes, we shouldn't be passing in `auth` as text, nor comparing / splitting it with `str` constants.

Comment (tseaver) [reply src/AccessControl/userfolder.py]: I've opened #56 to track this issue.

Comment (icemac) [reply src/AccessControl/userfolder.py]: @tseaver Thank you.

Comment (dataflake) [reply src/AccessControl/userfolder.py]: Fixed in https://github.com/zopefoundation/AccessControl/commit/ad391da2cceb9d286d824c457479112e8743f74a

**Pull request merged**